experiments/benchmarks/land.jl

```diff
@@ -364,7 +364,7 @@ function setup_prob(t0, tf, Δt; nelements = (101, 15))
     τ_NIR_leaf = FT(0.25)
 
     # Energy Balance model
-    ac_canopy = FT(2.5e5)
+    ac_canopy = FT(2.5e4)
```

Comment:
I think it would be nice to keep the parameters corresponding to the setup we want to be able to run, and adjust the timesteppping as needed. It might help us have a better idea of how close we are to where we want to be

Reply:
I agree! Ill revert back

Reply:
ah, our target is actually 2.5e3. I had made it bigger to run in the first place. I may need to lower dt.